mapit_mysociety_org/templates/account/signup.html

```diff
@@ -21,12 +22,17 @@
             <a href="mailto:{{ CONTACT_EMAIL }}">get in touch</a> so we
             can discuss how we can provide you with the service you require.
         </p>
+
         <form id="signup_form" method="post" action="{% url "account_signup" %}" autocapitalize="off" {% if form.is_multipart %} enctype="multipart/form-data"{% endif %}>
             {% csrf_token %}
             {% include 'account/_form_fields.html' %}
             <div class="account-form__field account-form__field--submit">
-                <button type="submit" class="btn">{% trans "Sign up" %}</button>
+                <button id="customButton" class="btn btn--primary">{% trans "Sign up" %}</button>
```

Comment:
We auto-generate the username, so phew! But anyway, Stripe doesn't take the money client-side, it just stores the card details and provides a token, it's only when that token is sent to Stripe server side (which we do in `form_valid`) that anything would be charged.